examples/benchmarks/tablebench/.mocharc.customBenchmarks.cjs

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Mocha configuration file to run memory-profiling tests
+ */
+"use strict";
+
+const getFluidTestMochaConfig = require("@fluid-internal/mocha-test-setup/mocharc-common");
+
+const packageDir = __dirname;
+const baseConfig = getFluidTestMochaConfig(packageDir);
+
+const nodeOptions =
+	baseConfig["node-option"] !== undefined
+		? Array.isArray(baseConfig["node-option"])
+			? baseConfig["node-option"]
+			: [baseConfig["node-option"]] // If string, wrap as array to use spread operator
+		: []; // If undefined, use an empty array
+
+nodeOptions.push("expose-gc", "gc-global", "unhandled-rejections=strict");
```

Comment:
Note to myself: we should probably re-evaluate these for custom benchmarks. Execution time and memory usage ones do need it because they explicitly invoke garbage collection, but we don't need to do that for custom benchmarks. Not entirely sure what `unhandled-rejections` does. @jikim-msft can you also capture a work item to revisit these settings, and removing them from custom benchmark configs if we deem they're not actually necessary for that scenario?

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/12926

Item created.